_episodes/05-starting-jekyll.md

```diff
@@ -7,75 +7,351 @@ questions:
+    ## Cite us
+    You can cite the project as:  
+    
+    >    *My research project. Zenodo. https://zenodo.org/record/doi*
+    
+    ## Contact us
+   
+    Email: [team@my.research.org](mailto:{% raw %}{{ site.email }}{% endraw %})   
+    
+    Twitter: [@my_research_project](https://twitter.com/my_research_project)   
+    ~~~
+    
+3. Go to your website to see the changes.
+4. Note that site parameters will not render nicely when viewing files in GitHub (they will be displayed as text
+`{% raw %}{{ site.PARAMETER_NAME }}{% endraw %}` rather than the parameter's rendered value) but will in the website.
+
+> ## Exercise 
+> In `about.md` we have a Twitter URL under the 'Contact us' section. That's one piece of information that could go into 
+> global parameters in `_config.yml` as you may want to repeat it on a footer of every page. 
+> Make changes to your website to extract Twitter URL as a global parameter.
 > > ## Solution
+> > 1. Add parameter twitter to `_config.yml`: 
 > >
-> > We will create the `pi` variable by adding the following line to the yaml header.
-> > ~~~
-> > pi: "Ada Lovelace"
-> > ~~~
+> >    ~~~
+> >    title: "Building Websites in GitHub"
+> >    description: "This research project develops training materials for reseachers wanting to learn to build project
+> >    websites in GitHub with GitHub Pages."
+> >    email: "team@my.research.org"
+> >    twitter: "https://twitter.com/my_research_project"
+> >    ~~~
+> >    {: .language-yaml}
+> > 2. Make use of the twitter parameter in `about.md`: 
 > >
-> > Here is an example of using it in the `index.md` file
-> > ~~~
-> > ---
-> > pi: "Ada Lovelace"
-> > ---
+> >    ~~~  
+> >    # About
+> >   
+> >    ## Project
+> >    {% raw %}{{ site.description }}{% endraw %}
+> >           
+> >    ## Funders
+> >    We gratefully acknowledge funding from the XYZ Founding Council, under grant number 'abc'.
+> >   
+> >    ## Cite us
+> >    You can cite the project as:  
+> >    
+> >    >    *My research project. Zenodo. https://zenodo.org/record/doi*
+> >    
+> >    ## Contact us
+> >   
+> >    Email: [team@my.research.org](mailto:{% raw %}{{ site.email }}{% endraw %})   
+> >    
+> >    Twitter: [@my_research_project]({% raw %}{{ site.twitter }}{% endraw %})   
+> >    ~~~
+> >    
+> > 3. Note that you should not see any changes to your website really. However, you can now access your Twitter URL from
+> > any website page, should you need to.    
+> {: .solution}
+{: .challenge}
+
+> ## Reuse and Reduce
+> Jekyll's global parameters are a useful way to keep all your site-wide configuration in 
+> a single place (even if you only use them once). In combination with Jekyll layouts/templates (to be covered in the next episode) they are a great way of creating reusable markup snippets that can be repeated on multiple or even on every page of your website. Reuse helps you reduce the amount of code you have to write.  
+{: .callout}
+
+### List Parameters and For Loops
```

Comment:
I wonder if, in the end, this section will be better moved into a later episode in the lesson: this episode is quite long and I suspect loops and dictionaries might be a good fit elsewhere as we write the remaining content. But this is well-explained and I'm totally fine with leaving it here for now 🙂

Reply:
I agree, I feel like the "Reuse and Reduce" section is a nice ending to this episode.

Reply:
Yup, lists and parameters feel a bit jumpy and not a really good fit under Global Parameters - they also fit in Page Parameters section. I will just copy+paste into the next lesson and then we can work from there.

Reply:
Fixed via 3cb2063.